electrumsv/daemon.py

```diff
@@ -126,8 +128,10 @@ def get_rpc_credentials(config: SimpleConfig) -> Tuple[Optional[str], Optional[s
         rpc_password = to_string(pw_b64, 'ascii')
         config.set_key('rpcuser', rpc_user)
         config.set_key('rpcpassword', rpc_password, save=True)
-    elif rpc_password == '':
+    elif rpc_password == '' and not is_restapi:
```

Comment:
Can you change both these messages to be still warnings but less alarming. As it is, we get people who can't work out why things are not working, but assume this is the reason. Maybe something like "No password has been set for the REST API, and any users who want to will be able to login."